pontoon/api/tests/test_views.py

```diff
@@ -0,0 +1,615 @@
+            "strings_with_warnings": p.strings_with_warnings,
+            "strings_with_errors": p.strings_with_errors,
+            "missing_strings": p.missing_strings,
+            "unreviewed_strings": p.unreviewed_strings,
+            "complete": p.complete,
+        }
+        for p in sorted(
+            Project.objects.filter(system_project=True)
+            | Project.objects.filter(disabled=False, system_project=False),
+            key=lambda p: p.pk,
+        )
+    ]
+
+    for project in response.data["results"]:
+        project.pop("locales", None)
+        project.pop("tags", None)
+
+    results = sorted(response.data["results"], key=lambda p: p["slug"])
+    expected_results = sorted(expected_results, key=lambda p: p["slug"])
+
+    # includes Terminology, Tutorial and 3 system projects
+    assert response.data["count"] == 5
+    assert results == expected_results
+
+
+@pytest.mark.django_db
+def test_disabled_projects(
+    django_assert_num_queries,
+):
+    ProjectFactory.create_batch(3, disabled=True)
+    ProjectFactory.create_batch(3, system_project=True)
+    with django_assert_num_queries(4):
+        response = APIClient().get("/api/v2/projects/?include_disabled")
+
+        assert response.status_code == 200
+
+    expected_results = [
+        {
+            "slug": p.slug,
+            "name": p.name,
+            "priority": p.priority,
+            "deadline": p.deadline,
+            "visibility": p.visibility,
+            "contact": p.contact,
+            "info": p.info,
+            "system_project": p.system_project,
+            "disabled": p.disabled,
+            "sync_disabled": p.sync_disabled,
+            "pretranslation_enabled": p.pretranslation_enabled,
+            "total_strings": p.total_strings,
+            "approved_strings": p.approved_strings,
+            "pretranslated_strings": p.pretranslated_strings,
+            "strings_with_warnings": p.strings_with_warnings,
+            "strings_with_errors": p.strings_with_errors,
+            "missing_strings": p.missing_strings,
+            "unreviewed_strings": p.unreviewed_strings,
+            "complete": p.complete,
+        }
+        for p in sorted(
+            Project.objects.filter(disabled=True)
+            | Project.objects.filter(disabled=False, system_project=False),
+            key=lambda p: p.pk,
+        )
+    ]
+
+    for project in response.data["results"]:
+        project.pop("locales", None)
+        project.pop("tags", None)
+
+    results = sorted(response.data["results"], key=lambda p: p["slug"])
+    expected_results = sorted(expected_results, key=lambda p: p["slug"])
+
+    # includes Terminology and 3 other disabled projects
+    assert response.data["count"] == 4
+    assert results == expected_results
+
+
+@pytest.mark.django_db
+def test_project_translation_stats():
+    print(Project.objects.all())
+
+
+@pytest.mark.django_db
+def test_project_locale():
```

Comment:
I'd create and tests some stats here, too. And also verify the number of DB queries.